includes/Modules/AdSense.php

```diff
@@ -1124,4 +1131,20 @@ protected function reset_analytics_adsense_linked_settings() {
 			)
 		);
 	}
+
+	/**
+	 * Resets the Ad Blocking Recovery notification.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function reset_ad_blocking_recovery_notification() {
+		$dismissed_prompts = ( new Dismissed_Prompts( $this->user_options ) );
+
+		$current_dismissals = $dismissed_prompts->get();
+
+		if ( isset( $current_dismissals['ad-blocking-recovery-notification'] ) && $current_dismissals['ad-blocking-recovery-notification']['count'] < 3 ) {
+			$dismissed_prompts->remove( 'ad-blocking-recovery-notification' );
+		}
+	}
+
```

Comment:
@hussain-t this is the logic for resting the notification.

Reply:
Yes, it works when I disconnect but not when resetting the plugin.